### ISSUE TITLE
convert ui.browseableMessage to wx.html2.WebView

### ISSUE THREAD

**Issue by LeonarddeR** [nvaccess/nvda#18878]
### Is your feature request related to a problem? Please describe.
NVDA shows browseable messages in a legacy Internet Explorer control.

### Describe the solution you'd like
Consider using `wx.html2.WebView` instead.
This webview uses Edge Webview 2 by default on modern Windows with an automatic fallback to IE when necessary.

### Describe alternatives you've considered
Leave as is

### Additional context
Current accessibility issues with wx webview should be solved by https://github.com/nvaccess/nvda/pull/18877.

**Comment by seanbudd** [comment]
When converting this over, we'd need thorough testing to ensure JS/CSS/HTML we feed into the dialog work across browsers

**Comment by LeonarddeR** [comment]
I don't think we need to feed js anymore. The buttons can become real wx buttons.

**Comment by Adriani90** [comment]
Would WX webview also allow displaying tables correctly in e.g. OCR recognition window? If yes, I would vote for at least increasing the priority to p4, because converting would open up significant opportunities for better accessibility in displaying information for NVDA users.
